engines/dreamweb/stubs.cpp

```diff
@@ -1261,6 +1277,9 @@ void DreamWebEngine::copyName(uint8 type, uint8 index, uint8 *dst) {
 }
 
 void DreamWebEngine::commandWithOb(uint8 command, uint8 type, uint8 index) {
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	Common::String theText = "";
```

Comment:
The initialization is redundant. Common::String is already initialized with an empty string upon creation